qt/pool.cpp

```diff
@@ -66,7 +66,13 @@ Pool::~Pool()
 
 bool Pool::load()
 {
-    return as_pool_load (d->m_pool, NULL, NULL);
+    GError* error = nullptr;
```

Comment:
Ideally use `g_autoptr` here:
`g_autoptr(GError) error = nullptr;`
(then you can get rid of the `g_error_free` line)